ledger/src/blockstore_processor.rs

```diff
@@ -1963,10 +1959,6 @@ fn load_frozen_forks(
             m.stop();
```

Comment:
I suppose the difference in the new code is that the `all_banks.insert()` and the process_single_slot timer both have to wait for rewards sending to complete. I assume this difference should be fairly trivial...?

Reply:
> I suppose the difference in the new code is that the all_banks.insert() and the process_single_slot timer both have to wait for rewards sending to complete

Technically yes. However, the logic of getting the rewards from a `Bank` has shifted:
- Tip of master currently gets rewards from `Bank` and sends rewards across the channel
- This PR retrieves rewards from the received `Bank` in the recording thread

Given these and the fact that `CacheBlockMetaService` was going to send a `Bank` in `process_single_slot()` anyways, this PR does not introduce any extra work inside the `process_single_slot` timer. But, this PR does reduce the overall amount of work for this replay thread form two crossbeam channel sends to one crossbeam channel send. This reduction in number of sends should indeed be trivial